res/css/views/right_panel/_TimelineCard.scss

```diff
@@ -84,6 +87,13 @@ limitations under the License.
             }
         }
 
+        &[data-layout=irc] {
+            .mx_EventTile_avatar,
+            .mx_MessageTimestamp {
+                position: absolute;
```

Comment:
The position of timestamp on group layout has been specified with [`_EventTile.scss`](https://github.com/matrix-org/matrix-react-sdk/blob/68c94c6400938cca6f5d663574695dcb867713cc/res/css/views/rooms/_EventTile.scss#L119).